### ISSUE TITLE
Add a model

### ISSUE THREAD

**Issue by shirounanashi** [karaokenerds/python-audio-separator#58]
Can you please add and host this model? It's a VR Arc model, and it's the best model I know for keeping and removing backing vocals. It works on UVR5
https://drive.google.com/file/d/1LSHlTvVt4FJxG8VGetueAuNlDc5w3Deu/view?usp=sharing

**Comment by beveradb** [comment]
Sorry, I'm not sure how this actually works with UVR - could you show me some screenshots / give me some instructions to explain how you currently use it with UVR?

If so, and if I'm able to get it working myself with UVR, I might be able to add support for that model - but at the moment, the output from `audio-separator -l` lists all supported models, and those are supported only because I'm able to extract the model configuration data from UVR itself (well, from the repositories on github which UVR uses to host config data).

If the model you've linked is legitimately supported by UVR, it must have configuration data hosted somewhere (potentially mapped using an MD5 hash of the model file) - in order to support it in `audio-separator`, I need to be able to find and access that model config data.

Hope that makes sense!
- Andrew

**Comment by shirounanashi** [comment]
I simply added the folder "C:\Ultimate Vocal Remover\models\VR_Models". Regarding the config data, i found this in the file "C:\Ultimate Vocal Remover\models\VR_Models\model_data\model_data.json"

```
    "f6ea8473ff86017b5ebd586ccacf156b": {
        "vr_model_param": "4band_v2_sn",
        "primary_stem": "Instrumental",
        "is_karaoke": true
    },
```

**Comment by shirounanashi** [comment]
@beveradb Now I have another request!
Can you add the BSRoformer models that were added in the latest UVR5 beta update?

**Comment by shirounanashi** [comment]
By the way, I managed to make the model I sent here work in Audio Separator, I just had to change the model check file and it worked without problems

**Comment by beveradb** [comment]
FYI @shirounanashi I've now released `audio-separator` version `0.17.1` with support for the `BS-RoFormer` and `Mel-Band RoFormer` models, e.g. `model_mel_band_roformer_ep_3005_sdr_11.4360.ckpt`

I highly recommend checking them out, the mel-roformer model performs better than pretty much everything else I've tried so far!

As for hosting/adding custom models, I'm not personally motivated to add support for that, but you're welcome to raise a pull request if you want to add that support yourself 😄
